front_end/core/host/RNPerfMetrics.ts

```diff
@@ -5,26 +5,21 @@
 
 export type RNReliabilityEventListener = (event: ReactNativeChromeDevToolsEvent) => void;
 
-type UnsunscribeFn = () => void;
-export type RNPerfMetrics = {
-  addEventListener: (listener: RNReliabilityEventListener) => UnsunscribeFn,
-  removeAllEventListeners: () => void,
-  sendEvent: (event: ReactNativeChromeDevToolsEvent) => void,
-};
-
 let instance: RNPerfMetrics|null = null;
 
 export function getInstance(): RNPerfMetrics {
   if (instance === null) {
-    instance = new RNPerfMetricsImpl();
+    instance = new RNPerfMetrics();
   }
   return instance;
 }
 
-class RNPerfMetricsImpl implements RNPerfMetrics {
+type UnsunscribeFn = () => void;
+class RNPerfMetrics {
   #listeners: Set<RNReliabilityEventListener> = new Set();
+  #launchId: string|null = null;
 
-  addEventListener(listener: RNReliabilityEventListener): () => void {
+  addEventListener(listener: RNReliabilityEventListener): UnsunscribeFn {
```

Comment:
Nit: Let's fix the typo?
```suggestion
  addEventListener(listener: RNReliabilityEventListener): UnsubscribeFn {
```